### PR TITLE
Export list of donors

### PR DESCRIPTION
**Pull request by malted**
This PR closes https://github.com/hackclub/hcb/issues/6610 and adds a provisionally admin-only button to export a set of the donor emails for an organisation.

image

https://hackclub.slack.com/files/U03DFNYGPCN/F077ALX5TS8/image.png
See, this is hardly a CSV.

### CODE REVIEW

File: app/views/events/donation_overview.html.erb [hunk 1]
```diff
@@ -126,8 +126,12 @@
...
             <div class="menu__content h5" data-behavior="menu_content">
-            <%= link_to "Export donations (CSV)", export_donations_path(event: @event, format: :csv), target: :_blank %>
-            <%= link_to "Export donations (JSON)", export_donations_path(event: @event, format: :json), target: :_blank %>
+              <%= link_to "Export donations (CSV)", export_donations_path(event: @event, format: :csv), target: :_blank %>
+              <%= link_to "Export donations (JSON)", export_donations_path(event: @event, format: :json), target: :_blank %>
+              <% admin_tool do %>
+                <%= link_to "Export donor emails (CSV)", export_donations_path(event: @event, format: :csv, only: [:email]), target: :_blank %>
+                <%= link_to "Export donor names & emails (CSV)", export_donations_path(event: @event, format: :csv, only: [:name, :email]), target: :_blank %>
+              <% end %>
             </div>
```
Comment (sampoder) [inline app/views/events/donation_overview.html.erb]: I think we can make these available to non-admins. We could also just call it "Export donors (CSV)".

Comment (malted) [reply app/views/events/donation_overview.html.erb]: Done.

File: app/policies/donation_policy.rb [hunk 2]
```diff
@@ -21,6 +21,10 @@ def export?
     record.event.users.include?(user) || user&.admin?
   end
 
+  def export_donors?
+    true
```
Comment (garyhtou) [inline app/policies/donation_policy.rb]: ‼️

**Review by garyhtou — DISMISSED** [review]

File: app/policies/donation_policy.rb (hunk 2 above)
Comment (malted) [reply app/policies/donation_policy.rb]: `WONTFIX`

Comment (sampoder) [reply app/policies/donation_policy.rb]:
```suggestion
    record.event.users.include?(user) || user&.admin?
```

**Review by sampoder — DISMISSED** [review]

**Review by sampoder — APPROVED** [review]

**Pull request merged**